### ISSUE TITLE
Unable to install "MungeSumStats" for R version in .yml echoR Conda environment

### ISSUE THREAD

**Issue by Keaty19** [RajLabMSSM/echolocatoR#89]
Hi guys! Currently trying to get started with the package - using the robust installation method of using the echoR Conda environment by using the .yml file, however when I get to the stage of installing "MungeStats" in the R console whilst in the echoR environment I get the error:

```
Warning message:
package ‘MungeStats’ is not available for Bioconductor version '3.12'
```

And when trying to update BiocManager to a later version so "MungeStats" is available, I get the error:

```
> BiocManager::install(version="3.13")
Error: Bioconductor version '3.13' requires R version '4.1'; use
  `BiocManager::install(version = '3.12')` with R version 4.0; see
  https://bioconductor.org/install
```

Is there something I have missed or do I need to upgrade the R version on the echoR environment, and if so, what would be the easiest way to do this?

Thanks a lot!

Sam K

**Comment by bschilder** [comment]
Hey @Keaty19, there's a couple of issues going on here.

1. First, `MungeStats` is not an R package that I'm aware of, but `MungeSumStats` is.
2. `MungeSumStats` was first released in Bioc version 3.14. So you need to have at least that version installed (I'd recommend simply using the current release version, 3.15).
3. More recent versions of Bioc require more recent versions of R (eg R v4.2). However the conda distribution of R is very behind, making it impossible to install the latest release of Bioc.

You can circumvent all of these complications by using a Docker container. `echolocatoR` does not yet have its own dedicated Docker container, so I recommend using the `MAGMA.Celltyping` Docker container for now, and then installing `echolocatoR` within it:
https://neurogenomics.github.io/MAGMA_Celltyping/articles/docker

I will be releasing an `echolocatoR`-dedicated Docker container once the echvoerse branch is ready to be released.

**Comment by Keaty19** [comment]
Hi Brian,

Doh! You can tell I posted that at the end of a long week. Apologies - I meant "MungeSumstats" which throws up the same error as i'm unable to update BiocManager in the Conda environment.

I've taken your advice and used the MAGMA.Celltyping Docker container instead and everything is installing effectively now. Thanks a lot for your help!

Sam K
